code/api/src/config/database.json

```diff
@@ -14,5 +14,13 @@
     "host": "127.0.0.1",
     "dialect": "postgres",
     "seederStorage": "sequelize"
+  },
```

Comment:
Love that we are writing in a testing database environment! I had a lot of fun working through the issues and I'm glad we have decent notes from the setup experience :)